src/config/index.ts

```diff
@@ -67,6 +67,7 @@ export const SOLD_LAND_FILTER_PAGE = "sold-land-filter";
 export const TRUST_INFO_PAGE = "trust-information";
 export const USE_PAPER_PAGE = "use-paper";
 export const WHO_IS_MAKING_FILING_PAGE = "who-is-making-filing";
+export const OVERSEAS_NAME_PAGE = "overseas-name";
```

Comment:
Would have probably called this constant ENTITY_NAME_PAGE (even though URL differs) and then used 'entity name' in all the code artefacts but not a problem.

Reply:
it does seem that the prototype page is called overseas-name so it is in keeping with the prototype ( i think our other page names follow this too)